### ISSUE TITLE
Mouse Gestures Interact with the Active Window Weirdly

### ISSUE THREAD

**Issue by DEATHB4DEFEAT** [horriblename/hyprgrass#87]
I set the example mouse binds to move/resize windows and when moving my terminal it thinks I'm click&dragging all over it but also moves the window as expected, resizing works fine. When using the move window with Firefox in Wayland mode it thinks I'm doing the pinch zoom and moves me around on the page while also moving the window as expected, resizing usually works fine but while writing this it made my mouse scroll the page to wherever I put it and I can't reproduce that. Other apps with pinch gestures (like Dolphin file manager) also have a similar reaction to Firefox, scaling a lot when moving around.
edit: "foot" terminal doesn't seem to have this issue, maybe it's just X11 apps being funky as they do

Here's my current hyprgrass config file with the comments removed if that's helpful
```py
plugin:touch_gestures {
  sensitivity = 2.0
  long_press_delay = 400
}

bind = , edge:r:l, workspace, +1
bind = , edge:l:r, workspace, -1

bind = , edge:l:d, exec, pactl set-sink-volume @DEFAULT_SINK@ -5%
bind = , edge:l:u, exec, pactl set-sink-volume @DEFAULT_SINK@ +5%

bind = , swipe:4:d, killactive

###
bindm = , longpress:2, movewindow
bindm = , longpress:3, resizewindow
###

bind = , edge:d:u, exec, $runner

bind = , edge:u:d, togglespecialworkspace, magic
bind = , swipe:3:u, movetoworkspace, special:magic
bind = , swipe:3:d, exec, ~/.config/hypr/scripts/dt_plugins_hyprgrass_mv-win-active-workspace.sh
```

Video of what I mean

https://github.com/horriblename/hyprgrass/assets/77995199/e154a6c0-71df-4b2c-b803-310b91d41219

**Comment by horriblename** [comment]
I definitely did not handle mouse binds correctly. This part of hyprland code is particularly convoluted so I'm not surprised

**Comment by nine7nine** [comment]
@DEATHB4DEFEAT 

I noticed this too -- one workaround is to swap the them:

# longpress can trigger mouse binds:
bindm = , longpress:3, movewindow
bindm = , longpress:2, resizewindow

this way you don't get all of the pinch to zoom issues when moving a window. then for resize - just avoid pinch to zoom (moving your finders closer/apart) when resizing the window. ~> You'll still get some jankyness / resizing issues (in window contents), just due to hyprland's own behavior, but still;  it's a bit better., IMO... 

This seemed to work alright for me / more manageable.

**Comment by horriblename** [comment]
this should be fixed in 0.7 (4d9495ccd122)

**Comment by horriblename** [comment]
let me know if it still doesn't work
